### ISSUE TITLE
llTargetOmega can not work in Group Prims?

### ISSUE THREAD

**Issue by toweiyg** [WhiteCoreSim/WhiteCore-Dev#270]
In 0.9.5,  I build a group model.  In the root prim, i add the script:
`default
{
    state_entry()
    {
        llTargetOmega(<0,0,1>,0.1, 0.0);
    }
}`

I hope the whole model will cycle with the root prim. But it not work.

If i just put the up code in a single prim, the prim will cycle.

**Comment by emperorstarfinder** [comment]
Good morning,

This is the link to llTargetOmega info on the Secondlife Scripting Wiki.  Looks like you might actually be missing a piece in your script:  http://wiki.secondlife.com/wiki/LlTargetOmega

That being said if that still doesn't work then at that point llTargetOmega will need to be reviewed.

**Comment by toweiyg** [comment]
Thank you @emperorstarfinder ,but the same code and same prims can work under  opensim0.7.6

**Comment by emperorstarfinder** [comment]
I will take a look at the implementation and see if there is something missing.  It could very well be that llTargetOmega is not fully implemented.

Also it should be noted that WhiteCore does use the old DotNetEngine and not XEngine so there may be functions that are not correctly working or are just not implemented yet.

**Comment by fly-man-** [comment]
llTargetOmega is 75% implemented, the basics is in but there's 14 commits from OpenSim that weren't whitecorified which makes the implementation not complete. The problem was that those are based on XEngine and we're still on DNE

**Comment by emperorstarfinder** [comment]
Yes this appears to be the case.  I'll see if I cannot implement it.

It appears to be implemented in the LSL_Api but might need to implement the pieces in other areas of the code such as SceneObjectPart.  Ill need to look at which 14 commits from Opensim aren't committed to see what is actually missing.
